catalog/solr/solr-app/src/main/resources/features.xml

```diff
@@ -30,22 +30,22 @@
 
     <feature name="catalog-solr-provider" install="auto" version="${project.version}"
              description="Catalog Provider to interface with Solr ${solr.version}">
-        <feature prerequisite="true">solr-app</feature>
         <bundle>mvn:ddf.catalog.solr/catalog-solr-provider/${project.version}</bundle>
     </feature>
 
-	<feature name="solr-app" install="auto" version="${project.version}"
-             description="The Solr Catalog Provider (SCP) is an implementation of the CatalogProvider interface using Apache Solr ${solr.version} as a data store.\nThe SCP supports extensible metacards, fast/simple contextual searching, indexes xml attributes/CDATA sections/XML text elements, contains full XPath support, works with an embedded local Solr Server, and also works with a standalone Solr Server.::Solr Catalog">
-		<feature prerequisite="true">catalog-app</feature>
-
-        <bundle>mvn:org.codice.thirdparty/jts/${jts.bundle.version}</bundle>
-	</feature>
-
     <feature name="catalog-solr-solrclient" install="auto" version="${project.version}"
              description="Configure Solr parameters.">
         <feature prerequisite="true">catalog-solr-provider</feature>
         <bundle>mvn:ddf.catalog.solr/catalog-solr-solrclient/${project.version}</bundle>
     </feature>
 
+    <feature name="solr-app" install="auto" version="${project.version}"
+             description="The Solr Catalog Provider (SCP) is an implementation of the CatalogProvider interface using Apache Solr ${solr.version} as a data store.\nThe SCP supports extensible metacards, fast/simple contextual searching, indexes xml attributes/CDATA sections/XML text elements, contains full XPath support, works with an embedded local Solr Server, and also works with a standalone Solr Server.::Solr Catalog">
+        <feature prerequisite="true">catalog-app</feature>
```

Comment:
:question: I noticed we remove prerequisite="true" in a lot of places.  Do we still need it here?

Reply:
I only removed the ones that were causing cyclical dependencies. I'm concerned removing any additional ones will cause additional feature cleanup to happen in this PR.